tests/pytorch/test_pytorch_model_export.py

```diff
@@ -425,6 +439,28 @@ def test_model_log_persists_specified_conda_env_in_mlflow_model_directory(
     assert saved_conda_env_text == pytorch_custom_env_text
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_model_log_persists_requirements_in_mlflow_model_directory(
+    sequential_model, pytorch_custom_env
+):
+    artifact_path = "model"
+    with mlflow.start_run():
+        mlflow.pytorch.log_model(
+            pytorch_model=sequential_model,
+            artifact_path=artifact_path,
+            conda_env=pytorch_custom_env,
+        )
+        model_path = _download_artifact_from_uri(
+            "runs:/{run_id}/{artifact_path}".format(
+                run_id=mlflow.active_run().info.run_id, artifact_path=artifact_path
+            )
+        )
+
+    saved_pip_req_path = os.path.join(model_path, "requirements.txt")
+    _compare_conda_env_requirements(pytorch_custom_env, saved_pip_req_path)
+
+
```

Comment:
Could we add a Pytorch-specific test here that verifies that if an explicitly passed in pip requirements file is passed in that we respect that instead?